src/verilog.y

```diff
@@ -5785,14 +5787,30 @@ combinational_body<nodep>:      // IEEE: combinational_body + sequential_body
                 yTABLE tableEntryList yENDTABLE         { $$ = new AstUdpTable{$1, $2}; }
         ;
 
-tableEntryList<udpTableLinep>:  // IEEE: { combinational_entry | sequential_entry }
+tableEntryList<udpTableLinep>:  // IEEE: { combinational_entry }
                 tableEntry                              { $$ = $1; }
         |       tableEntryList tableEntry               { $$ = addNextNull($1, $2); }
         ;
 
-tableEntry<udpTableLinep>:      // IEEE: combinational_entry + sequential_entry
-                yaTABLELINE                             { $$ = new AstUdpTableLine{$<fl>1, *$1}; }
-        |       error                                   { $$ = nullptr; }
+tableEntry<udpTableLinep>:      // IEEE: combinational_entry
+                tableLine                               { $$ = $1; }
+        ;
+
+tableLine<udpTableLinep>:
```

Comment:
I thnk tableEntry is not need as a rule, can call tableLine directly (update the caller's rule).  And it does handle sequential too.

```suggestion
tableLine<udpTableLinep>:       // IEEE: combinational_entry + sequential_entry
```